src/client/urlTokenBaseHTTPClient.ts

```diff
@@ -110,7 +113,7 @@ export class URLTokenBaseHTTPClient implements BaseHTTPClient {
       return;
     }
 
-    let body: Uint8Array | null = null;
+    let body = new Uint8Array();
```

Comment:
second allocation at line 120 below

Reply:
This is a bit odd, I improved it in 6874e43